src/js/auth.js

```diff
@@ -0,0 +1,41 @@
+const { nanoid } = 'nanoid';
+
+const auth = () => {
```

Comment:
This function should have:

  - `getLocalStorageUser` which accepts a `user_email` as a parameter and returns the associated session from the local storage if exists, else it should return `null`.
  - `addLocalStorageUser` which accepts a `sessionData` as a parameter and stores it inside the user array local storage.